rust/storage/src/lib.rs

```diff
@@ -38,14 +40,28 @@ impl ChromaError for GetError {
     fn code(&self) -> ErrorCodes {
         match self {
             GetError::NoSuchKey(_) => ErrorCodes::NotFound,
+            GetError::ObjectStoreError(_) => ErrorCodes::Internal,
             GetError::S3Error(_) => ErrorCodes::Internal,
             GetError::LocalError(_) => ErrorCodes::Internal,
         }
     }
 }
 
+impl From<::object_store::Error> for GetError {
+    fn from(e: ::object_store::Error) -> Self {
```

Comment:
I would prefer we did this less broadly than wrap all object store errors, the point of the facade is to present a unified error tree.

https://docs.rs/object_store/latest/object_store/enum.Error.html. 

Many of these can be unbundled / deduplicated and coded differently.

Reply:
I unbundled the one kind of error we currently support.  Would you like me to unbundle and make a facade for the whole error?